README.md

```diff
@@ -2,36 +2,50 @@
 
 ## Overview
 
-A brief description. Be clear about the purpose of the resolver,
-its capabilities and limitations.
+A custom filter for Sceptre for unquoting resolvers appearing in var files.
 
 ## Installation
 
 Installation instructions
 
 To install directly from PyPI
 ```shell
-pip install sceptre-resolver-template
+pip install jinja-unquote-resolvers-filter
 ```
 
 To install from the git repo
 ```shell
-pip install git+https://github.com/Sceptre/sceptre-resolver-template.git
+pip install git+https://github.com/Sceptre/jinja-unquote-resolvers-filter.git
 ```
 
 ## Usage/Examples
 
-Refer to the [Sceptre resolver docs](https://docs.sceptre-project.org/latest/docs/resolvers.html#resolvers)
+Refer to the [Sceptre resolver docs](https://docs.sceptre-project.org/latest/docs/filters.html#filters) <!-- This page does not exist yet. -->
 for more information.
 
+In your var file:
+
 ```yaml
-parameter|sceptre_user_data:
-  <name>: !custom <args>
+Subnets:
+  - '!stack_output_external mystack::subnet_a'  # Quotes needed to protect a YAML tag.
+  - '!stack_output_external mystack::subnet_b'
+
+VPC: '!stack_output_external mystack::vpc_id'
 ```
 
-## Example
+In your config:
 
 ```yaml
-parameters:
-  message: !custom "HelloWorld"
+j2_environment:
+  extensions:
+    - jinja_unquote_resolvers_filer.UnquoteResolversFilterExtension
```

Comment:
jinja_unquote_resolvers_filer/jinja_unquote_resolvers_filter

Also shouldn't this follow the `<dir>.<class>` pattern?, if so shouldn't it be `filter.UnquoteResolversFilterExtension`?

Reply:
I think this will install into a dir like

```
/usr/local/lib/python3.10/site-packages/jinja_unquote_resolvers_filter 
```

And in your `j2_environment` you would have

```yaml
j2_environment:
  extensions:
    - jinja_unquote_resolvers_filter.UnquoteResolversFilterExtension
```

So I think it's right.

Reply:
On testing this I confirmed this is right.